x-pack/plugins/security_solution/public/dashboards/pages/details/index.tsx

```diff
@@ -52,8 +52,8 @@ const DashboardViewComponent: React.FC = () => {
     canReadDashboard ? null : DashboardViewPromptState.NoReadPermission
   );
   const [dashboardDetails, setDashboardDetails] = useState<DashboardDetails>();
-  const onDashboardContainerLoaded = useCallback((dashboardContainer: DashboardContainer) => {
-    const dashboardTitle = dashboardContainer.getTitle().trim();
+  const onDashboardContainerLoaded = useCallback((dashboard: DashboardAPI) => {
+    const dashboardTitle = dashboard.getTitle().trim();
```

Comment:
```suggestion
    const dashboardTitle = dashboard?.getTitle().trim();
```

Reply:
Found that `dashboard` might be null sometimes
<img width="2559" alt="Screenshot 2023-04-13 at 10 58 59" src="https://user-images.githubusercontent.com/6295984/231731288-18e1bc6c-13c2-4b32-ae16-58b949e0b030.png">